pkg/services/ngalert/notifier/channels/email.go

```diff
@@ -81,8 +78,8 @@ func (en *EmailNotifier) Notify(ctx context.Context, as ...*types.Alert) (bool,
 				"CommonLabels":      data.CommonLabels,
 				"CommonAnnotations": data.CommonAnnotations,
 				"ExternalURL":       data.ExternalURL,
-				"RuleUrl":           "TODO",
-				"AlertPageUrl":      "TODO",
+				"RuleUrl":           path.Join(en.appURL, "/alerting/list"),
+				"AlertPageUrl":      path.Join(en.appURL, "/alerting/list?alertState=firing&view=state"),
```

Comment:
I'm not sure this is the way to go here, is there a page in the frontend that would allow us to filter by a set of labels? If so, I'd pretty much prefer that one than trying to sleuth through the list.

Reply:
I like it; it seems nice to have if it works and easy to fix if not.